src/koans/02_strings.clj

```diff
@@ -3,68 +3,68 @@
             [clojure.string :as string]))
 
 (meditations
-  "A string is nothing more than text surrounded by double quotes"
-  (= __ "hello")
+ "A string is nothing more than text surrounded by double quotes"
+ (= (str 'hello)  "hello")
 
-  "But double quotes are just magic on top of something deeper"
-  (= __ (str 'world))
+ "But double quotes are just magic on top of something deeper"
+ (= "world" (str 'world))
 
-  "You can do more than create strings, you can put them together"
-  (= "Cool right?" (str __ __))
+ "You can do more than create strings, you can put them together"
+ (= "Cool right?" (str "Cool " "right?"))
 
-  "You can even get certain characters"
-  (= \C (get "Characters" __))
+ "You can even get certain characters"
+ (= \C (get "Characters" 0))
 
-  "Or even count the characters"
-  (= __ (count "Hello World"))
+ "Or even count the characters"
+ (= 11 (count "Hello World"))
 
-  "But strings and characters are not the same"
-  (= __ (= \c "c"))
+ "But strings and characters are not the same"
+ (= false (= \c "c"))
 
-  "What if you only wanted to get part of a string?"
-  (= "World" (subs "Hello World" __ __))
+ "What if you only wanted to get part of a string?"
+ (= "World" (subs "Hello World" 6 11))
 
-  "How about joining together elements in a list?"
-  (= __ (string/join '(1 2 3)))
+ "How about joining together elements in a list?"
+ (= "123" (clojure.string/join '(1 2 3)))
```

Comment:
You have required `clojure.string` as `string` in the namespace, so you don't need the `clojure.` prefix.